### DISCUSSION TITLE
How to notify admins about vulnerabilities detected by trivy?

### DISCUSSION THREAD

**Discussion by rithvik-tw** [aquasecurity/trivy#2733] [Q&A] [answered]
Hi,

We are running trivy as a cron job to detect OS level vulnerabilities. The cron job is working fine but we want to alert the admin when a vulnerability is detected by trivy. Is there any plugin that I can install with trivy or a set of instructions that we can follow to configure alerting. 

Thanks,
Rithvik

**Comment by knqyf263** [comment]
I think you can use Postee. @simar7 
https://github.com/aquasecurity/postee

**Comment by simar7** [comment, marked as answer]
hi @rithvik-tw - you could use the newly released [Trivy Webhook Plugin](https://github.com/aquasecurity/trivy-plugin-webhook) and send results from Trivy to [Postee](https://github.com/aquasecurity/postee). 

Postee can then send alerts (via email, JIRA etc.) to you. Let us know if you need any help onboarding on Postee. You can find the documentation for Postee here: https://aquasecurity.github.io/postee/v2.7.2/ – I plan on writing a section on how to send Trivy results into Postee using the webhook plugin as part of the documentation shortly. Issue to track that is here: https://github.com/aquasecurity/postee/issues/443

**Comment by rithvik-tw** [comment]
Hi @simar7 & @knqyf263 , Thanks. I have gone through the documentation and it seemed helpful. Will give this a try.

Reference of the documentation for others :) 
https://aquasecurity.github.io/postee/v2.8.2/blueprints/trivy-vulnerability-scan/
